instaclustr/Common_vpc_peering.go

```diff
@@ -0,0 +1,71 @@
+package instaclustr
```

Comment:
Should rename this file to be `common_vpc_peering.go` to maintain file name consistency